README.md

```diff
@@ -133,6 +133,30 @@ shutdown mockyeah's Express server. Failing to do so will likely result in
 `EADDRINUSE` exceptions. This is due to mockyeah attempting to start a server on
 an occupied port.
 
+### Configuration File
+`.mockyeah` default configurations
+
+```json
+{
+  "name": "mockyeah",
+  "host": "localhost",
+  "port": 4001,
+  "fixturesDir": "./mockyeah/fixtures"
+}
+```
+
+`name`: Used to identify the origin of logged output
+
+`host`: The host mockyeah will run on
+
+`port`: The port mockyeah will run on
+
+`fixturesDir`: The relative path to the fixtures directory
```

Comment:
This might be phrased better as: "The relative path to the fixtures direction from the `.mockyeah` file location"... or something to that effect. We should test to verify this to be true.